### ISSUE TITLE
Embed Webpage Widget Failed

### ISSUE THREAD

**Issue by HippoPhileus** [OlegWock/anori#198]
I just set up Anori today, so let me know if I'm just missing a permissions check or other setting, but when I click on the embedded webpage widget I get an error. 

<img width="1440" alt="Screen Shot 2024-07-03 at 3 23 46 PM" src="https://github.com/OlegWock/anori/assets/174653033/4dca08c0-7e81-4e70-8e82-0241b0cc1f59">

Is this something on my end/anyone know how to fix this?

Thank you!
HippoPhileus

**Comment by OlegWock** [comment]
It sometimes just doesn't work ¯\_(ツ)_/¯
Problem seem to be on Chrome side, so I can't fix this in extension. But if you disable and then enable back Anori, it should fix the problem for a while. You can do this on chrome://extensions page (you might need to toggle it a few times, if it won't work after first one)

**Comment by 8227846265** [comment]
I'm getting the same issue for TickTick. Error log says:

Refused to frame 'https://ticktick.com/' because an ancestor violates the following Content Security Policy directive: "frame-ancestors 'self' https://www.notion.so https://*.notion.site notion://www.notion.so https://*.google.com https://www.wolai.com chrome-extension://phhdjikfindbnmghmegdcbcgdpoogacl chrome-extension://diankknpkndanachmlckaikddgcehkod extension://phhdjikfindbnmghmegdcbcgdpoogacl extension://diankknpkndanachmlckaikddgcehkod chrome-extension://mdkekgdakdomdpefbfibhjimhinfgfkb extension://mdkekgdakdomdpefbfibhjimhinfgfkb".

**Comment by OlegWock** [comment]
Yeah, that's the error. Try re-enabling extension, it usually helps

**Comment by OlegWock** [comment]
Hello, I'm doing issues cleanup in this repo. I'll be closing this issue because it's pretty old and it's more of a browser limitation/issue
